src/pages/admin/ReceivePayment.vue

```diff
@@ -40,27 +40,40 @@ export default {
   components: {QRCodeScanner},
   computed: {
     ...mapGetters('admin', ['getIsDialogOpenned', 'getReceived', 'getIsDisabled']),
+    isDisabled() {
```

Comment:
bad function naming ....
name a function follow the real purpose of the function
like validateNumberInput ...